### PR TITLE
Distinguish zero bar from others in event histogram

### PR DESCRIPTION
**Pull request by denamwangi**
histogram_bottom_figurelght

UPDATE: Bottom bar screenshot

### CODE REVIEW

**Comment by MaxBittker** [comment]
addresses https://github.com/getsentry/sentry/issues/5241

File: src/sentry/static/sentry/app/components/stackedBarChart.jsx [hunk 1]
```diff
@@ -172,6 +173,11 @@ const StackedBarChart = React.createClass({
     return maxval;
   },
 
+  barColorClass(pct, idx) {
+    const colClass = this.props.barClasses[idx];
+    return pct === 0 ? classNames(colClass, 'zero-bar') : colClass;
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/components/stackedBarChart.jsx]: I think you should use the classNames predicate syntax here, like 
```
return classNames(colClass, {'zero-bar':  pct === 0})
```

File: src/sentry/static/sentry/less/stream.less [hunk 2]
```diff
@@ -541,10 +541,18 @@
...
+
   .event-graph {
-    .sparkline a {
+    .sparkline, .barchart, a {
       height: 24px;
     }
+    .zero-bar {
+      background-color: transparent;
+    }
+    .barchart {
```
Comment (MaxBittker) [inline src/sentry/static/sentry/less/stream.less]: is this to draw the line underneath it? why this and not `border-bottom: 1px solid @gray-lightest`?

**Review by MaxBittker — CHANGES_REQUESTED** [review]
looks good! only blocking this is dropping the ternary because i think this reads more cleanly.

File: src/sentry/static/sentry/app/components/stackedBarChart.jsx [hunk 3]
```diff
@@ -172,6 +173,11 @@ const StackedBarChart = React.createClass({
     return maxval;
   },
 
+  barColorClass(pct, idx) {
+    const colClass = this.props.barClasses[idx];
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/components/stackedBarChart.jsx]: could you help me understand what this barClasses array is?

**Comment by benvinegar** [comment]
Hey folks, we re-visiting this?

**Comment by denamwangi** [comment]
I definitely want to-i'll plan to loop back around to it next week

**Pull request closed without merging**